pkg/api/apis/operators/v1alpha1/installplan_types.go

```diff
@@ -65,10 +66,12 @@ const (
 type StepStatus string
 
 const (
-	StepStatusUnknown    StepStatus = "Unknown"
-	StepStatusNotPresent StepStatus = "NotPresent"
-	StepStatusPresent    StepStatus = "Present"
-	StepStatusCreated    StepStatus = "Created"
+	StepStatusUnknown             StepStatus = "Unknown"
+	StepStatusNotPresent          StepStatus = "NotPresent"
+	StepStatusPresent             StepStatus = "Present"
+	StepStatusCreated             StepStatus = "Created"
+	StepStatusCreatedCRD          StepStatus = "CreatedCRD"
```

Comment:
this seems odd, since `CRD` is already in the step kind. What about something like "Pending" or "WaitingForApi" or "Registering" (or something better that you think of)?